app/models/manageiq/providers/azure/cloud_manager/provision/options_helper.rb

```diff
@@ -6,4 +6,8 @@ def root_password
   def resource_group
     @resource_group ||= ResourceGroup.find_by(:id => options[:resource_group])
   end
+
+  def security_group
+    @security_group ||= SecurityGroup.find_by(:id => get_option(:security_groups))
```

Comment:
@bronaghs I was wrong about the control in the current version.  While it looks like a drop-down compared to the old control it does allow multiple selections and is working properly for the other cloud providers.

We just need to identify if there is a way to use the control as a single select for Azure.  I am not aware of any properties we can set on the dialog field that would control that.

@dclarizio  Any idea?